resources/asciidoctor/lib/edit_me/extension.rb

```diff
@@ -28,50 +28,73 @@ def process(document)
       url = url[0..-2] if url.end_with? '/'
       edit_urls << { toplevel: toplevel, url: url }
     end
-    edit_urls.reverse!
+    # Prefer the longest matching edit url
+    edit_urls = edit_urls.sort_by { |e| [-e[:toplevel].length, e[:toplevel]] }
```

Comment:
you could also use [max](https://ruby-doc.org/core-2.6.3/Enumerable.html#method-i-max), using the length as the comparable attribute in the supplied block.

Reply:
I really want to sort the whole list so the longest are first so we check them first.

Reply:
ah, ok. I didn't see the big picture - you want a sorted list, not just the longest. Nevermind!